notebooks/python-notebook.ipynb

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
Is this just a demo file?  Or will all users use this notebook by default?

Reply:
Demo file for the example workflow under tests